sparkbox/designsystemssurvey
# DSS-6 feat: add favicon into gatsby manifest

### Description
Add favicon to project, then to manifest. [Manifest plugin](https://www.gatsbyjs.org/packages/gatsby-plugin-manifest/) takes care of generating favicons of different sizes.

### To Test
Check in the `<head>` in dev tools that favicons are being generated. Check that you can see it in the tab.
